getting_started_materials/intro_tutorials_and_guides/11_Introduction_to_Strings.ipynb

```diff
@@ -0,0 +1,3040 @@
+{
```

Comment:
<p>In notebook cells, the <strong>repr</strong> only shows up in the cell output if it's the final execution. It would be good to remove these lines for clarity across the notebook (<code>pandasSeries.astype("string")</code>. If the intent is to show them, they should be printed or in their own cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/notebooks-contrib/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Resolved</p>